### ISSUE TITLE
Fix fluentsharp test that is failing at appveyor

### ISSUE THREAD

**Issue by DinisCruz** [o2platform/FluentSharp#57]
https://ci.appveyor.com/project/DinisCruz/fluentsharp

![image](https://cloud.githubusercontent.com/assets/656739/5792262/9985ad2e-9f07-11e4-83f7-db5aea7b995c.png)

https://ci.appveyor.com/project/DinisCruz/fluentsharp/build/tests

![image](https://cloud.githubusercontent.com/assets/656739/5792265/ae5d8d7a-9f07-11e4-9cf0-b543fbf0d674.png)

**Comment by DinisCruz** [comment]
it's a different one now

![image](https://cloud.githubusercontent.com/assets/656739/5801652/7a0ff3da-9fe5-11e4-9443-80bf155ee733.png)

this one is caused by the fact that the tests are running without a desktop

**Comment by alexandrustanimir** [comment]
I was unable to find a method to check for this kind of execution context where you don`t have desktop :(. As a workaround  we can ignore the test if we find an environment variable specific to appveyor.

var envs = Environment.GetEnvironmentVariables().toList<DictionaryEntry>();
            foreach (var item in envs)
            {
                if (item.Value.ToString().contains("appveyor"))
                { 
                    "Appveyor is running in terminal mode.(Skipping)".assert_Ignore();
                    return;
                }
            }
What do you think ?

**Comment by DinisCruz** [comment]
yeah, with the caveat that we should be using an extension method like

```
if ("appveyor".environmentVariable().valid())
    ...
```

:)

Btw, have you pinged Appveyor and asked about tests that require desktop? They really should have a solution for that, or no UI QA tests will be able to be executed there
